asterisk-mixin/mixin.libsonnet

```diff
@@ -1,7 +1,15 @@
+local config = (import 'config.libsonnet');
+
 {
-  grafanaDashboards: {
+  grafanaDashboards: 
+  if config._config.enableLokiLogs then
+  {
+    'asterisk-overview.json': (import 'dashboards/asterisk-overview.json'),
+    'asterisk-logs.json': (import 'dashboards/asterisk-logs.json')  
```

Comment:
Minor nitpick/question here, as I don't recall seeing this pattern before.

Do we have a style of trending towards clearly distinct conditional blocks like here, where overview is imported in both conditions?

Asking because my gut feel is that the conditional should only affect the logs dashboard? Or maybe this is a quirk of jsonnet making this easier?

Reply:
It is more of a jsonnet syntax thing, I was not able to do it another way unfortunately. We normally define these at the dashboard level, but these are json dashboards so it does not accept jsonnet logic.

Reply:
Got it, thank you for explaining :D